### ISSUE TITLE
Directions are not weighting trails heavily

### ISSUE THREAD

**Issue by nickrogers** [azavea/cac-tripplanner#409]
I've been having trouble with getting the route to snap to a trail, particularly if I am selecting "safer" as my route option. Here, you can see the line of the trail is parallel to my route, and yet I'm not being to get on and off of it. Can "safer" routes (or routes in general, but particularly safer routes) weigh more heavily towards using a bike facility?

![image](https://cloud.githubusercontent.com/assets/12200440/8384033/2bf0ce10-1c0c-11e5-8e0e-2e1581bf2133.png)

**Comment by nickrogers** [comment]
another example: 
![image](https://cloud.githubusercontent.com/assets/12200440/8384207/85ff7d74-1c0d-11e5-9e3f-e98361793717.png)

**Comment by jbranigan** [comment]
In the first case, the Schuylkill River Trail is not marked as a path west of where it crossed the river. So the direction turn off in that section, since OpenStreetMap thinks it's only an abandoned railroad.

In the second case, the length of trail east of the lake is denoted as a gravel foot path. This actually steers OpenTripPlanner away from recommending that path for cycling. When starting nearer to the Cooper River Trail, the cycling directions use it.

Strange directions related to OpenStreetMap data will likely be discovered elsewhere, but this is a community-built, ever-improving resource for data. To contribute, create an account at OpenStreetMap.org

**Comment by nickrogers** [comment]
What is weird about the SRT example is that the directions place you on the trail - so it knows it is a trail, and then off the trail, then back on, etc. on both sides of the river. 

I just noticed the second example has another issue - it's actually going to the wrong place! that address should be the Environmental Center we went to for that meeting. Also, it looks like there is a path suitable for biking on the street view, but I haven't actually been there so I can't say for sure. As a general guideline, can we toggle whether to recommend gravel paths on and off? We may discover that most of the users have hybrid or wider tired bikes that are perfectly fine on gravel paths, and might want to include them.

Thanks for the suggestion on creating account at OpenStreetMap, John. I imagine I may do more harm than good there, given my experience with mapping software! We'll continue to flag errors on the site's directions, and it would be great if the software team can check into them to determine the root of the problem, so we know whether it is software based or OpenStreetMap based.
